cmd/create.go

```diff
@@ -41,6 +41,15 @@ var createCmd = &cobra.Command{
 			log.Error("Unable to create container: " + err.Error())
 			os.Exit(1)
 		}
+
+		// Store the LXC Image -> Container relationship
+		log.Running("Storing image container relation ")
+		err = lxclient.SetContainerTemplateRealation(name, template, true)
```

Comment:
the lxclient package should be self-contained and have only code that talks to the lxc/lxd services.  This function `SetContainerTemplateRelation` is part of lxdev, and should be in the cmd package in a new file.